lib/evercookie.rb

```diff
@@ -45,44 +45,46 @@ def self.get_namespace
 
   # getter for session key variable for set action
   def self.hash_name_for_set
-    "#{@@host_url}#{@@hash_name}_set".to_sym
+    "#{@@host_url}/#{@@hash_name}_set".to_sym
   end
 
   # getter for session key variable for get action
   def self.hash_name_for_get
-    "#{@@host_url}#{@@hash_name}_get".to_sym
+    "#{@@host_url}/#{@@hash_name}_get".to_sym
   end
 
   # getter for session key variable all stored evercookies
   def self.hash_name_for_saved
-    "#{@@host_url}#{@@hash_name}_saved".to_sym
+    "#{@@host_url}/#{@@hash_name}_saved".to_sym
   end
 
+  # TODO: Can we create cookies that are accessible across multiple domains (i.e. domain arg)(i.e. across marketing sites)
```

Comment:
you can only create cookies under the domain the code runs on, but if it uses the top level domain (.weddingwire.com) then it would be accessible from all subdomains